qa/test-harness/requirements.txt

```diff
@@ -1,3 +1,4 @@
+git+https://github.com/usdot-jpo-ode/ode-output-validator-library@master#egg=odevalidator
```

Comment:
I think we should keep the submodule as the linkage even though the config issue is resolved. It is consistent with the rest of ODE organizational structure and makes it easier to update both repos locally.

Reply:
I think there are advantages to both methods, and both methods require a pip install command. Since the user is already required to do a --recurse-submodules to get the ODE itself working, we know that they will be guaranteed to have the code locally.

If and when we move the library to be hosted on PyPI, then I believe we should switch over to managing it there. For now, having it as a submodule is fine.